x-pack/plugins/actions/server/mocks.ts

```diff
@@ -26,7 +31,19 @@ const createStartMock = () => {
   return mock;
 };
 
+const createServicesMock = () => {
+  const mock: jest.Mocked<Services & {
```

Comment:
Shouldn't this just be `jest.Mocked< AlertServices>` ? 🤔

Reply:
The `jest.Mocked<...>` doesn't apply recursively. This was necessary to allow `services.savedObjectsClient.get.mockResolvedValue(...)`.